src/unitxt/inference.py

```diff
@@ -1553,6 +1572,32 @@ def get_return_object(self, predict_result, response, return_meta_data):
         return predict_result
 
 
+class VLLMRemoteInferenceEngine(OpenAiInferenceEngine):
+    label: str = "vllm"
+
+
+class RITSInferenceEngine(OpenAiInferenceEngine):
+    label: str = "rits"
+
+    def get_default_headers(self):
+        return {"RITS_API_KEY": self.credentials["api_key"]}
```

Comment:
why do we need this as an header? It seems strange as we already pass the api key

Reply:
tested it, and it seem we need it. Strange.